src/pseudopeople/filter.py

```diff
@@ -1,14 +1,14 @@
 from dataclasses import dataclass
-from typing import Optional, Union
+from typing import Union
 
 import pandas as pd
 
 
 @dataclass
 class DataFilter:
-    column_name: Optional[str]
+    column_name: str
```

Comment:
look closely at these changes. I'm reasonably sure that `column_name` really can't be None. I think the reason it was orignally Optional is b/c this DataFilter gets instantiated often with a state_column_name and not all Datasets have one, but we only create a filter by Datasets who do.

Reply:
That seems plausible to me. I remember something like that when I did my big type checking PR last sprint.